eslint.shared.config.mjs

```diff
@@ -1,10 +1,9 @@
-import globals from 'globals';
+import o3rTemplate from '@o3r/eslint-config/template';
```

Comment:
I don't understand why it does not go next to `import o3rConfig from '@o3r/eslint-config';` but if it is automatic now we don't really need to review the order, everyone will have the same